Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,112 @@
+Parser:
+  Title: Network Session ASIM parser for Cisco Meraki
+  Version: '1.0'
+  LastUpdated: May 20, 2022
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.2'
+References:
+  - Title: ASIM Network Session Schema
+    Link: https://aka.ms/ASimNetworkSessionDoc
+  - Title: ASIM
+    Link: https://aka.ms/AboutASIM
+  - Title: Organization Application Security Events 
+    Link: https://developer.cisco.com/meraki/api-latest/#!get-organization-appliance-security-events
+Description: |
+  This ASIM parser supports normalizing Cisco Meraki IDS Events produced by the Microsoft Sentinel Cisco Meraki Security Events connector to the ASIM Network Session normalized schema.
+ParserName: vimNetworkSessionCiscoMeraki                                  
+EquivalentBuiltInParser: _Im_NetworkSession_CiscoMeraki                                  
+ParserParams:
+  - Name: starttime
```

Comment:
Did you (manually) test the filters?

Reply:
I just performed tests on time filters. Will complete the testing for the remaining filters as well.
Will keep the thread updated.

Reply:
Completed the tests. All the filters are working as expected. Anything to be tested in particular?

Reply:
Should have outlined my thoughts around testing before you started, but I am sure you did a great job. Anyways, the idea is to run the parser for each parameter with a value that exists in the data and see that only the records matching the value are returned. As an approximation, for example to enable automation, checking that the count of records when using the filter is not 0 and not the total number of records is also good.

Reply:
Got it, I have verified the filters and I do see the variation in count as well. Thank you so much for the advise.